README.md

```diff
@@ -178,6 +178,11 @@ This project uses melos to manage all the packages inside this repo.
 Install melos: `dart pub global activate melos`
 Setup melos to point to the dependencies in your local folder: `melos bootstrap`
 
+## Android
+
+Library requires kotlin version `1.5.31`.
+
```

Comment:
please add also instructions how to bump the Kotlin version. Not everyone working with flutter has android knowledge.

Reply:
Add instructions